src/main/java/com/strandls/document/service/Impl/DocumentServiceModule.java

```diff
@@ -16,6 +16,7 @@
 	@Override
 	protected void configure() {
 		bind(DocumentService.class).to(DocumentServiceImpl.class).in(Scopes.SINGLETON);
+		bind(DocumentListServiceImpl.class).in(Scopes.SINGLETON);
```

Comment:
bind the interface to the implementation to a singleton, the interface should only be accessed from outside , not the implemenation , so that the local function in the implementation cannot be accessed from others